packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -667,6 +676,22 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         return _getFeeAdjustedCollateral(rawTotalPositionCollateral);
     }
 
+    /**
+     * @notice Accessor method to calculate a transformed price using the provided finanicalProductLibrary specified
+     * during contract deployment. If no library was provided then no modification to the price is done.
+     * @param price input price to be transformed.
+     * @return transformedPrice price with the transformation function applied to it.
+     */
+
+    function transformPrice(FixedPoint.Unsigned memory price) public view returns (FixedPoint.Unsigned memory) {
+        if (address(financialProductLibrary) == address(0)) return price;
+        try financialProductLibrary.transformPrice(price) returns (FixedPoint.Unsigned memory transformedPrice) {
```

Comment:
One question that I don't have a strong opinion on: should we _require_ that the user provides a valid financial product library and remove this branching logic (even if they just use a default one that does no transform)?

Reply:
We could do this, especially if we have a "default" public one that simply returns the price with no transformation logic applied. However, this amounts to the exact same thing that this PR implements, but adds a higher barrier as now you need to know this address. Also, this would require us to validate the library at emp deployment.

Curious to hear others thoughts @nicholaspai

Reply:
I'd side with Chris' current implementation and just assume zero-address = no transformation precisely because we don't want users to have to know which address the default library is deployed at. 

The only downside of this is the 1 extra line that performs the `if (x == address(0) ...` check, I think.